packages/core/src/app/checkout/CheckoutRootNodeId.ts

```diff
@@ -0,0 +1 @@
+export const CHECKOUT_ROOT_NODE_ID = 'checkout-app' as string;
```

Comment:
Do we need to add this assertion? TS compiler should be able to infer the type correctly 🤔

Reply:
Yes, it's redundant. I have removed it.